src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs

```diff
@@ -61,7 +61,7 @@ internal partial class Controller :
             _autoBraceCompletionChars = autoBraceCompletionChars;
             _isDebugger = isDebugger;
             _isImmediateWindow = isImmediateWindow;
-            _roles = textView.Roles.ToImmutableHashSet();
+            _roles = ImmutableHashSet.Create<string>(textView.TextBuffer.ContentType.TypeName);
```

Comment:
Should this be renamed to `_contentTypes` then? Also, was this just accidental crud left around from another edit?

Reply:
Oh, I suspect this is accidental: this is why the integration tests are breaking.

Reply:
@rchande This was in the original work. Do you know if it's a necessary change or should be backed out?

Reply:
I don't think this is required.

Reply:
:arrow_right: Fixed in edd0bf3534e6ee325b6de1e0ebffc8811d811faa.